src/selectors/getRelativeSources.js

```diff
@@ -7,20 +7,22 @@
 import { getProjectDirectoryRoot, getSources } from "../selectors";
 import { chain } from "lodash";
 import type { Source, RelativeSource } from "../types";
-import { getSourcePath } from "../utils/source";
+import { getURL } from "../utils/sources-tree";
 import { createSelector } from "reselect";
 
-function getRelativeUrl(url, root) {
+function getRelativeUrl(source, root) {
+  const { group, path } = getURL(source);
   if (!root) {
-    return getSourcePath(url);
+    return path;
```

Comment:
re-computing the source path can be expensive. It is better to use `getURL` which is cached